deploy/src/main/resources-filtered/quick-start/docker-compose-with-httpproxy.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  autoscaler:
+    env_file:
+      - ./rabbitmq.env
+    environment:
+      DOCKER_HOST: ${DOCKER_HOST:-http://192.168.56.10:2375}
+      CAF_AUTOSCALER_MAXIMUM_INSTANCES: 4
+      CAF_AUTOSCALER_DOCKER_SWARM_STACK: jobservicedemo
+      CAF_DOCKER_SWARM_TIMEOUT: 30
+      HTTP_PROXY: ${HTTP_PROXY}
+      HTTPS_PROXY: ${HTTPS_PROXY}
+      NO_PROXY: ${NO_PROXY}
+    image: autoscaler/autoscale-dockerswarm-rabbit:{project.version}
```

Comment:
Note: Change the build name in the container pom to the above, not the compose file